Project.toml

```diff
@@ -11,10 +11,10 @@ Static = "aedffcd0-7271-4cad-89d0-dc628f76c6d3"
 
 [compat]
 IfElse = "0.1"
-LoopVectorization = "0.12.113"
+LoopVectorization = "0.12.159"
 PrecompileTools = "1"
-Static = "0.2 - 0.8"
-julia = "1.8"
+Static = "0.8"
+julia = "1.8, 1.9"
```

Comment:
Just for future reference, just "1.8" allows everything in the `1.x` series greater than 1.8 (since after 1.0, minor version numbers aren't breaking any more)